### PR TITLE
f-image-tile@v0.13.0 - Added Breakout images

### PR DESCRIPTION
**Pull request by AlixRichards**
Eventually the image tile will only support 'breakout' style images but at the moment I need to support both options. After this is fully rolled out I will be able to remove the standard option.

---

### Added
- Breakout image option and tests

---

Breakout style images
Screenshot 2022-05-30 at 15 57 04

Standard image tile
Screenshot 2022-05-30 at 15 57 34

## UI Review Checks

- [X] README and/or UI Documentation has been [updated]
- [X] Unit tests have been [created]
- [X] This code has been checked with regard to [our accessibility standards](http://fozzie.just-eat.com/documentation/general/accessibility/checklist)

## Browsers Tested

- [X] Chrome (latest)
- [X] Safari
- [X] Firefox

### CODE REVIEW

File: packages/components/atoms/f-image-tile/src/components/ImageTile.vue [hunk 1]
```diff
@@ -291,10 +373,18 @@ $image-tile-text-transform: translate3d(5px, 0, 0);
...
+    .c-imageTile--breakout & {
+        transform: translate3d(-spacing(b), 0, 0) scale(0.5) rotate(-60deg);
+    }
+
     .c-imageTile--selected & {
         @include image-tile-icon-selected-transform();
     }
 
+    .c-imageTile--selected.c-imageTile--breakout & {
+        @include image-tile-icon-selected-transform-breakout()
```
Comment (kevinrodrigues) [inline packages/components/atoms/f-image-tile/src/components/ImageTile.vue]: Minor, but missing a semicolon.

File: packages/components/atoms/f-image-tile/src/components/ImageTile.vue [hunk 2]
```diff
@@ -303,6 +393,10 @@ $image-tile-text-transform: translate3d(5px, 0, 0);
         .c-imageTile:hover & {
             @include image-tile-icon-selected-transform();
         }
+
+        .c-imageTile.c-imageTile--breakout:hover & {
+            @include image-tile-icon-selected-transform-breakout()
```
Comment (kevinrodrigues) [inline packages/components/atoms/f-image-tile/src/components/ImageTile.vue]: same.

File: packages/components/atoms/f-image-tile/src/components/ImageTile.vue [hunk 3]
```diff
@@ -323,14 +417,23 @@ $image-tile-text-transform: translate3d(5px, 0, 0);
...
+    .c-imageTile--selected.c-imageTile--breakout & {
+        transform: $image-tile-text-transform-breakout;
+    }
+
     @include media('>=mid') {
         .c-imageTile:hover & {
             transform: $image-tile-text-transform;
         }
+
+        .c-imageTile.c-imageTile--breakout:hover & {
```
Comment (kevinrodrigues) [inline packages/components/atoms/f-image-tile/src/components/ImageTile.vue:434]: Should these also cover :focus too? The one above too.

File: packages/components/atoms/f-image-tile/src/components/_tests/ImageTile.test.js [hunk 4]
```diff
@@ -169,6 +169,23 @@ describe('ImageTile', () => {
...
+
+                // Act
+                const wrapper = shallowMount(ImageTile, {
+                    propsData
+                });
+
+                const imageTileInnerWrapper = wrapper.find('[data-test-id="image-tile-inner-wrapper"]');
+
+                // Assert
+                expect(imageTileInnerWrapper.exists()).toBe(true);
```
Comment (kevinrodrigues) [inline packages/components/atoms/f-image-tile/src/components/_tests/ImageTile.test.js]: I'm not sure this expect matches what the description of the `it` states? Maybe the description needs updating to `should display the breakout image when the prop 'isBreakoutImage' is truthy`.

File: packages/components/atoms/f-image-tile/src/components/_tests/ImageTile.test.js [hunk 5]
```diff
@@ -169,6 +169,23 @@ describe('ImageTile', () => {
             });
         });
 
+        describe('isBreakoutImage :: ', () => {
```
Comment (kevinrodrigues) [inline packages/components/atoms/f-image-tile/src/components/_tests/ImageTile.test.js:172]: Do you also need a test for when this is falsey?

**Review by jamieomaguire — DISMISSED** [review]

File: packages/components/atoms/f-image-tile/src/components/_tests/ImageTile.test.js (hunk 5 above)
Comment (AlixRichards) [reply packages/components/atoms/f-image-tile/src/components/_tests/ImageTile.test.js:172]: I'll fix up the test above to do both 👍

File: packages/components/atoms/f-image-tile/src/components/ImageTile.vue (hunk 3 above)
Comment (AlixRichards) [reply packages/components/atoms/f-image-tile/src/components/ImageTile.vue:434]: The animation is only needed on hover, there are different focus styles 👍

**Review by jamieomaguire — APPROVED** [review]

**Review by 1itvinka — APPROVED** [review]

**Pull request merged**